pkg/lib/bundlev2/supported_resources.go

```diff
@@ -0,0 +1,57 @@
+package bundlev2
+
+import "fmt"
+
+const (
+	CSVKind                = "ClusterServiceVersion"
+	CRDKind                = "CustomResourceDefinition"
+	SecretKind             = "Secret"
+	ClusterRoleKind        = "ClusterRole"
+	ClusterRoleBindingKind = "ClusterRoleBinding"
+	ServiceAccountKind     = "ServiceAccount"
+	ServiceKind            = "Service"
+	RoleKind               = "Role"
+	RoleBindingKind        = "RoleBinding"
+	PrometheusRuleKind     = "PrometheusRule"
+	ServiceMonitorKind     = "ServiceMonitor"
+	RegistryV2CSVKind      = "Registry+v2/ClusterServiceVersion"
+	DeploymentKind         = "Deployment"
+	APIServiceKind         = "APIService"
+)
+
+type Namespaced bool
+
+// Add a list of supported resources to the map
+// Key: Kind name
+// Value: If namespaced kind, true. Otherwise, false
+var supportedResources = map[string]Namespaced{
+	CSVKind:                true,
+	CRDKind:                false,
+	ClusterRoleKind:        false,
+	ClusterRoleBindingKind: false,
+	ServiceKind:            true,
+	ServiceAccountKind:     true,
+	RoleKind:               true,
+	RoleBindingKind:        true,
+	PrometheusRuleKind:     true,
+	ServiceMonitorKind:     true,
+	RegistryV2CSVKind:      false,
+	DeploymentKind:         true,
+	APIServiceKind:         true,
+}
```

Comment:
Secret and ConfigMap are supported now as well

Reply:
I hope i am right to assume that they are not referenced in CSV

Reply:
added